packages/cli/src/lib/lifecycle-service.ts

```
@@ -47,6 +47,70 @@ export function getLifecycleLockFile(config: OrchestratorConfig, projectId: stri
+ * This closes the race in the naive force path (clearLifecycleLockFile →
+ * tryAcquireLifecycleLock) where the force worker could unlink a live peer's
+ * lock, causing both workers to proceed with overlapping locks. (orch-886k)
+ */
+export function forceLifecycleLock(
+  config: OrchestratorConfig,
+  projectId: string,
+): (() => void) | null {
+  const lockFile = getLifecycleLockFile(config, projectId);
+
+  // Step 1: read current lock owner PID.
+  let ownerPid: number | null = null;
```

ESLint

[error] This assigned value is not used in subsequent statements.

(no-useless-assignment)